ads/vendors/verizonmedia.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  writeScript,
+  validateData,
+} from '../../3p/3p';
+
+export function verizonmedia(global, data) {
```

Comment:
Can we correctly denote the types for these objects?

`global` is a `Window`.
`data` is `{{config: string}}`.

Additionally, this adds a new global for the `ads.externs.js` file `window.jacData`.

Reply:
updated the objects.  not sure what you want for the window.jacData.  Pushed the update.
thanks

Reply:
@ebrandmark2 Please add a line in https://github.com/ampproject/amphtml/blob/master/ads/ads.extern.js with

```
// verizonmedia.js
window.jacData
```

It helps the type check.